config/routes.rb

```diff
@@ -31,6 +31,7 @@
     resources :tags, only: %i[index show]
     root to: "home#index"
     resources :questions, only: %i[index new create show]
+    resources :attachments, only: %i[show]
```

Comment:
minor suggestion: here is just one action, maybe is not necessary to use %i notation 
```suggestion
    resources :attachments, only: :show
```

Reply:
Corrected. Thanks!